src/get-involved/index.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: Get Involved
+---
+
+There are many ways to get involved with Zinc collective and support our vision for a post-scarcity world. We hope you'll join us!
+
+### 👋🏻 Meet Cute
```

Comment:
I would avoid the romantic associations of "Meet Cute". Maybe just "meet" or "say hi" or "hang out" or a combination of those or...